uPortal-layout/uPortal-layout-core/src/main/java/org/apereo/portal/layout/dao/jpa/StylesheetUserPreferencesImpl.java

```diff
@@ -66,6 +67,7 @@
 class StylesheetUserPreferencesImpl implements IStylesheetUserPreferences {
     @Id
     @GeneratedValue(generator = "UP_SS_USER_PREF_GEN")
+    @Index(name = "IDX_SS_USER_PREF_ID")
```

Comment:
I am a bit confused about this one.  The `@Index` annotation is on the `id` member, which is the Hibernate-managed ID _and primary key_ column for the entity -- it definitely has an index already.

The foreign key index on `up_ss_user_pref_lay_attr(ss_user_pref_id)` also appears to be there, before the change...

![up_ss_user_pref_lay_attr](https://user-images.githubusercontent.com/1127174/35597072-fa7c67b2-05d9-11e8-8388-d581de66be17.png)

Reply:
Here is what this `@Index` annotation appears to do (in my environment)...

![up_ss_user_pref-after](https://user-images.githubusercontent.com/1127174/35597675-ed6046e0-05dc-11e8-9d12-56be01aed986.png)

It adds a `IDX_SS_USER_PREF_ID` index on the `SS_USER_PREF_ID` column of the `up_ss_user_pref` table.  (It does exactly what it looked like it did.)  Is that the intent?  (It isn't what I understood.)